sql/scan.go

```diff
@@ -120,6 +121,7 @@ type scanNode struct {
 	kvs              []client.KeyValue // the raw key/value pairs
 	kvIndex          int               // current index into the key/value pairs
 	rowIndex         int               // the index of the current row
```

Comment:
Is this field still being used? Looks like you replaced it with `renderRowIndex`.